### PR TITLE
 Add accountapi and txapi to nodejs demo

### PR DESCRIPTION
**Pull request by Yoruba**

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -692,7 +692,7 @@ This API sends the Theta/TFuel tokens. Note the API call can send either Theta t
 - tfuelwei: The amount of TFuel tokens to be sent (in TFuelWei, 1 TFuel = 10^18 TFuelWei)
 - fee: The transaction fee in TFuelWei
 - sequence: The expected sequence number of the from account
-- async: A boolean flag. If `async` is set to `false`, the RPC call will wait until the transaction has been included in a block, or a timeout reached. Otherwise, the RPC call will return immediately with the transaction hash.
```
Comment (jieyilong) [inline docs/api.md]: This is actually the correct behavior, could you revert? Thanks!

File: docs/mainnet.md [hunk 2]
```diff
@@ -18,19 +15,6 @@ Start a screen or tmux session, and use the following commands to launch a launc
...
-
-
-Use the following commands to launch and connect to the mainnet the next time.
-```
-export THETA_HOME=$GOPATH/src/github.com/thetatoken/theta
-cd $THETA_HOME
-theta start --config=../mainnet/walletnode
-```
-
+**Note**: when the mainnet Theta Node is launched for the first time, you need to choose a password to generate an address for the node. *Please choose a secure password and keep it in a safe place*. The next time when you launch the node again, you will need the password to unlock it.
```
Comment (jieyilong) [inline docs/mainnet.md]: Seems like this reverts the previous changes you made?

File: docs/api.md (hunk 1 above)
Comment (Yoruba) [reply docs/api.md]: @jieyilong sorry about that. In the new commit the changed are revert. If you want I can remove this pull requests and add it again.

File: docs/mainnet.md (hunk 2 above)
Comment (Yoruba) [reply docs/mainnet.md]: @jieyilong sorry about that. In the new commit this change is revert.

File: docs/api.md (hunk 1 above)
Comment (jieyilong) [reply docs/api.md]: cool thanks!

File: docs/mainnet.md (hunk 2 above)
Comment (jieyilong) [reply docs/mainnet.md]: thanks! we will update this document later

**Pull request merged**